index.bs

```diff
@@ -2482,6 +2526,34 @@ The <dfn attribute for="XRInputSourcesChangeEvent">added</dfn> attribute is a [=
 
 The <dfn attribute for="XRInputSourcesChangeEvent">removed</dfn> attribute is a [=/list=] of {{XRInputSource}}s that were removed from the {{XRSession}} at the time of the event.
 
+XRTrackedSourcesChangeEvent {#xrtrackedsourceschangeevent-interface}
```

Comment:
I don't remember if this was discussed at the face to face?
I think we need a different event otherwise existing experiences will start getting additional events that they won't know how to handle.

Reply:
To clarify: We can how a different _event_ but still have it use the same _interface_. The same way that we have a single `MouseEvent` interface that is provided for `mousedown`, `mouseup`, and `click` events. The proposal here is not to push everything into a single event, it's to re-use the data structure returned with the separate events for input vs. tracked sources.

Reply:
This issue is still outstanding. If you removed the `XRTrackedSourcesChangeEvent` and `XRTrackedSourcesChangeEventInit` and changed the `trackedsourceschange` event to use `XRInputSourcesChangeEvent` then I think this is mergable.